courses/machine_learning/deepdive2/feature_engineering/solutions/4_keras_adv_feat_eng.ipynb

```diff
@@ -0,0 +1,1751 @@
+{
```

Comment:
<p>as before. Use TODO comments to tie to numbered Learning objective above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>